meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -24,138 +24,407 @@ inherit mbl-artifact-names
 # will be used in your case.
 MBL_FLASH_ERASE_BLOCK_SIZE_KiB ?= "16384"
 
+MBL_DEFAULT_ALIGN_KiB ?= "${MBL_FLASH_ERASE_BLOCK_SIZE_KiB}"
+MBL_DEFAULT_FSTYPE ?= "ext4"
+MBL_DEFAULT_MOUNT_OPTS ?= "defaults"
+MBL_DEFAULT_FS_FREQ ?= "0"
+MBL_DEFAULT_FS_PASSNO ?= "0"
 
+# ------------------------------------------------------------------------------
+# Partition variables
+# ------------------------------------------------------------------------------
 # We have two different flavours of "partition" in MBL:
 # * Normal file system partitions, recorded in the partition table.
 # * Raw non-file system "partitions", not recorded in the partition table.
+# 
+# List of all partitions, in order
+MBL_PARTITION_NAMES = ""
+#
+# For each of these "partitions" there are the following variables:
+# * MBL_<partition>_ALIGN_KiB   - minimum alignment in KiB
+# * MBL_<partition>_SIZE_KiB    - size in KiB
+# * MBL_<partition>_SIZE_MiB    - size in MiB
+# * MBL_<partition>_OFFSET1_KiB - offset of first bank in KiB
+# * MBL_SKIP_<partition>        - skip this partition?
+# * MBL_<partition>_NO_FS       - is this a raw partition (not a file system partition)?
```

Comment:
I think we should remove the question mark.

Reply:
https://github.com/ARMmbed/meta-mbl/pull/617/commits/d70b2d4558985d027806009014b90261f4b894d8